source/core/DebugPoint.cpp

```diff
@@ -11,7 +11,12 @@ namespace Vireo {
     // Debug node which will check if breakpoint is set or not
     VIREO_FUNCTION_SIGNATURE1(DebugPoint, StringRef)
     {
-        return _NextInstruction();
+        #if kVireoOS_emscripten
+            #if GetDebugPointState(_Param(0))
```

Comment:
This cannot be a pre-processor directive right? This would need to be evaluated at run-time.

Reply:
Yeah. Changed this now.